test/hypothesis_utils.py

```diff
@@ -118,34 +133,23 @@ def array_shapes(draw, min_dims=1, max_dims=None, min_side=1, max_side=None):
     kH_range, kW_range: Ranges to generate kernel height and width. Must be
                         tuples of `(min, max)`
     max_groups: Maximum number of groups to generate
-    dtypes: A list of data types to generate. See note below.
 Generates:
     (X, w, b, g): Tensors of type `float32` of the following drawen shapes:
         X: (`nbatch, iChannels, H, W`)
         w: (`oChannels, iChannels // groups, kH, kW)
         b: `(oChannels,)`
         g: Number of groups the input is divided into
-    (scale, zero_point): Drawn from valid ranges derived from the dtypes
-    (qmin, qmax): Valid quantization ranges derived from the dtypes.
-    quantized_type: Data types for conversion in test.
-Note:
-    - The `dtypes` argument is used to infer the ranges. The elements should be
-    of length 1 or 2:
-        If the length is 1 -- The zero_point is not enforced.
-        If the length is 2 -- Argument 0 is torch dtype,
-            the zero_point is also enforced by argument 1
 """
 @st.composite
-def qtensors_conv(draw, min_batch=1, max_batch=3,
+def tensor_conv2d(draw,
```

Comment:
can we have float_min, float_max as arguments here as well?

Reply:
Also, does it make sense to support generating tensors with different formats? This always generates NCHW shape tensors. QNNPack requires inputs to be NHWC. It would be nice if this function could generate both.